L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -48,7 +57,9 @@ HGCalHistoSeedingImpl::Histogram HGCalHistoSeedingImpl::fillHistoClusters(
   for (int z_side : {-1, 1}) {
     for (int bin_R = 0; bin_R < int(nBinsRHisto_); bin_R++) {
       for (int bin_phi = 0; bin_phi < int(nBinsPhiHisto_); bin_phi++) {
-        histoClusters[{{z_side, bin_R, bin_phi}}] = 0;
+        histoClusters[{{z_side, bin_R, bin_phi}}].sumMipPt = 0;
```

Comment:
The goal was to avoid searching multiple times in the maps unnecessarily. The map should be accessed only once.